### ISSUE TITLE
V1.5.6 FFmpegFrameGrabber could not grab frames: avcodec_receive_frame() error -1094995529: Error during video decoding

### ISSUE THREAD

**Issue by luunn54** [bytedeco/javacv#1680]
Ver 1.5.6 (release version)

Issue: FFmpegFrameGrabber.grab could not grab frames

Test code:
```
        try {
            FFmpegLogCallback.set();

            FFmpegFrameGrabber g = new FFmpegFrameGrabber("attachment631-small.mp4");
            g.start();

            int count_frame = 0;
            while (true) {
                Frame frame = g.grab();
                if (frame == null) {
                    break;
                }

                count_frame++;
            }

            g.stop();
            System.out.println("Number of frames: " + count_frame);
        } catch (FFmpegFrameGrabber.Exception e) {
            e.printStackTrace();
        }
```

Output:
```
Warning: [h264 @ 0x7fbac2265800] SEI type 0 overread by 8 bits

Error: [h264 @ 0x7fbac2265800] no frame!

Info: Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'attachment631-small.mp4':

Info:   Metadata:

Info:     major_brand     : 
Info: isom
Info: 

Info:     minor_version   : 
Info: 512
Info: 

Info:     compatible_brands: 
Info: isomiso2avc1mp41
Info: 

Info:     encoder         : 
Info: Lavf57.34.103
Info: 

Info:   Duration: 
Info: 00:00:11.00
Info: , start: 
Info: 0.000000
Info: , bitrate: 
Info: 882 kb/s
Info: 

Info:   Stream #0:0
Info: (eng)
Info: : Video: h264 (High) (avc1 / 0x31637661), yuv420p, 720x480 [SAR 8:9 DAR 4:3], 879 kb/s
Info: , 
Info: 60 fps, 
Info: 60 tbr, 
Info: 19200 tbn, 
Info: 59.94 tbc
Info:  (default)
Info: 

Info:     Metadata:

Info:       handler_name    : 
Info: VideoHandler
Info: 

Info:       vendor_id       : 
Info: [0][0][0][0]
Info: 

Warning: [h264 @ 0x7fbac1a79600] SEI type 0 overread by 8 bits

Error: [h264 @ 0x7fbac1a79600] no frame!

Error: [h264 @ 0x7fbac1a75200] no frame!

Error: [h264 @ 0x7fbac1a75e00] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b18000] no frame!

Error: [h264 @ 0x7fbac1b18c00] no frame!

Error: [h264 @ 0x7fbac1b19800] no frame!

Error: [h264 @ 0x7fbac1a79c00] no frame!

Error: [h264 @ 0x7fbac1a75800] no frame!

Error: [h264 @ 0x7fbac1b15c00] no frame!

Error: [h264 @ 0x7fbac1b16800] no frame!

Error: [h264 @ 0x7fbac1b17400] no frame!

Error: [h264 @ 0x7fbac1b17a00] no frame!

Error: [h264 @ 0x7fbac1b18600] no frame!

Error: [h264 @ 0x7fbac1b19200] no frame!

Error: [h264 @ 0x7fbac1a79600] no frame!

Error: [h264 @ 0x7fbac1a75200] no frame!

Error: [h264 @ 0x7fbac1a75e00] no frame!

Error: [h264 @ 0x7fbac1b16200] no frame!

Error: [h264 @ 0x7fbac1b16e00] no frame!

Error: [h264 @ 0x7fbac1b17a00] no frame!

Error: [h264 @ 0x7fbac1b18600] no frame!

Error: [h264 @ 0x7fbac1b19200] no frame!

Error: [h264 @ 0x7fbac1a79600] no frame!

Error: [h264 @ 0x7fbac1a75200] no frame!

Error: [h264 @ 0x7fbac1a75e00] no frame!

Error: [h264 @ 0x7fbac1b16200] no frame!

Error: [h264 @ 0x7fbac1b16e00] no frame!

Error: [h264 @ 0x7fbac1b17a00] no frame!

Error: [h264 @ 0x7fbac1b18600] no frame!

Error: [h264 @ 0x7fbac1b19200] no frame!

Error: [h264 @ 0x7fbac1a79600] no frame!

Error: [h264 @ 0x7fbac1a75200] no frame!

Error: [h264 @ 0x7fbac1a75e00] no frame!

Error: [h264 @ 0x7fbac1b16200] no frame!

Error: [h264 @ 0x7fbac1b16e00] no frame!

Error: [h264 @ 0x7fbac1b17a00] no frame!

Error: [h264 @ 0x7fbac1b18600] no frame!

Error: [h264 @ 0x7fbac1b19200] no frame!

Error: [h264 @ 0x7fbac1a79600] no frame!

Error: [h264 @ 0x7fbac1a75200] no frame!

org.bytedeco.javacv.FFmpegFrameGrabber$Exception: avcodec_receive_frame() error -1094995529: Error during video decoding. (For more details, make sure FFmpegLogCallback.set() has been called.)
	at org.bytedeco.javacv.FFmpegFrameGrabber.grabFrame(FFmpegFrameGrabber.java:1388)
	at org.bytedeco.javacv.FFmpegFrameGrabber.grab(FFmpegFrameGrabber.java:1280)
	at abc.Test.main(Test.java:23)
```
OS: Mac and Centos

Test video:
[attachment631-small.mp4.zip](https://github.com/bytedeco/javacv/files/6931108/attachment631-small.mp4.zip)

**Comment by saudet** [comment]
Well, it says there's no frames. What is the result that you're expecting?

**Comment by luunn54** [comment]
there are 163 frames at the end of video - it work find with Ver 1.5.4. 
ignore that error solve the problem too

**Comment by saudet** [comment]
Please give it a try with JavaCV 1.5.7-SNAPSHOT: http://bytedeco.org/builds/

**Comment by luunn54** [comment]
this issue still persist on the snapshot build 1.5.7-20210805.084510-7.

I think we should ignore exceptions at:
https://github.com/bytedeco/javacv/blob/master/src/main/java/org/bytedeco/javacv/FFmpegFrameGrabber.java#L1388
https://github.com/bytedeco/javacv/blob/master/src/main/java/org/bytedeco/javacv/FFmpegFrameGrabber.java#L1427

**Comment by saudet** [comment]
I'm not sure about that. Could you check with JavaCV 1.5.5?

**Comment by luunn54** [comment]
it works fine with JavaCV 1.5.4 and 1.5.5 (no exception was thrown). But it broken on JavaCV 1.5.6 (and JavaCV 1.5.7-SNAPSHOT)
``
org.bytedeco.javacv.FFmpegFrameGrabber$Exception: avcodec_receive_frame() error -1094995529: Error during video decoding. (For more details, make sure FFmpegLogCallback.set() has been called.)
	at org.bytedeco.javacv.FFmpegFrameGrabber.grabFrame(FFmpegFrameGrabber.java:1388)
	at org.bytedeco.javacv.FFmpegFrameGrabber.grab(FFmpegFrameGrabber.java:1280)
``

**Comment by saudet** [comment]
You're right, it looks like it used to ignore those errors as well. I've fixed that in commit https://github.com/bytedeco/javacv/commit/551b00e5caf5c84eeeb4360b62d2310e28183fc4.

**Comment by luunn54** [comment]
This bug was fixed on build 1.5.7-20210805.124845-13. 
Thank you for your great support!

**Comment by saudet** [comment]
And released with JavaCV 1.5.7. Thanks to you for reporting these issues!
